.github/workflows/ci.yml

```diff
@@ -60,7 +60,10 @@ jobs:
         run: pip install -e .[rest]
 
       - name: Change Django Version for test
-        run: pip install -U ${{ matrix.django-version }}
+        run: |
+          pip install -U ${{ matrix.django-version }}
+          # TODO: Remove before merging
+          pip install --force-reinstall --no-cache-dir --no-deps --upgrade https://github.com/openwisp/openwisp-utils/tarball/admin-action-perm
```

Comment:
https://github.com/openwisp/openwisp-utils/pull/379 is merged, please update this.